rero_ils/modules/items/api.py

```diff
@@ -711,8 +711,7 @@ def item_location_retriever(cls, item_pid, **kwargs):
         #       to deal with multiple pickup locations for a library
         item = cls.get_record_by_pid(item_pid)
         if item:
-            library = item.get_library_of_last_location()
-            return library.get_pickup_location_pid()
+            return item.get_owning_pickup_location_pid()
```

Comment:
I recall that the `PO` asked to not rely on the the `pickup location` of the `owning library`. Instead he asked to find where the item is actually now by checking the `library` of the last transaction `loan`. This is a use case not documented in the Circulation state chart. The outcome, keep the two lines:
```
library = item.get_library_of_last_location()
return library.get_pickup_location_pid()
```